samples/net/coaps_server/prj_qemu_x86.conf

```diff
@@ -14,7 +14,7 @@ CONFIG_NET_PKT_TX_COUNT=14
 CONFIG_NET_BUF_RX_COUNT=30
 CONFIG_NET_BUF_TX_COUNT=30
 CONFIG_NET_IF_UNICAST_IPV6_ADDR_COUNT=3
-CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=2
+CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=5
```

Comment:
That is certainly possible, it is just difficult to know what the limit should be, but 3 could be enough for most uses. I can put this into my todo list unless anyone wants to send patch for this.